<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_set</class>
 <widget class="QWidget" name="Widget_set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(155, 187, 89, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(155, 187, 89, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(155, 187, 89, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(155, 187, 89, 255), stop:1 rgba(255, 255, 255, 255));
}
QRadioButton {
	padding: 1px;
}
QRadioButton::indicator:checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: rgba(246, 134, 86, 255);
	color: #a9b7c6;
	background-color:rgba(246, 134, 86, 255);
}
QRadioButton::indicator:!checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: rgb(246, 134, 86);
	color: #a9b7c6;
	background-color: transparent;
}
QWidget {
	background-color:#f0f0f0;
}
QLabel {
	color:rgb(17,17,17);
}
QWidget {
	color:rgb(0,0,0);
	background-color:rgb(76,93,99);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示上界：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_quit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_up_limit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>8000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_set_save">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示下界：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_down_limit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-3000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-4000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-5000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-6000</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>报警上界：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_alarm_uplimit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>7500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>报警下界：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_alarm_downlimit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>1500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-1500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-2500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-3000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-3500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-4000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-4500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-5000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-5500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-6000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-6500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-7000</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_set_filedelete">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>125</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>距离系数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_coefficient">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_demar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>125</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>更新系数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copyfile">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>125</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拷贝数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>灵山矿</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_set_quit</sender>
   <signal>clicked()</signal>
   <receiver>Widget_set</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
